osmhelper/osm_stops.py

```diff
@@ -33,123 +29,7 @@ def __repr__(self):
-
-        print self
-
-    def get_interim_stop_name(self):
-        api = overpy.Overpass()
-
-        result = api.query("""
-        <osm-script>
-          <query type="way">
-            <around lat="%s" lon="%s" radius="50.0"/>
-            <has-kv k="name" />
-            <has-kv modv="not" k="highway" v="trunk"/>
-            <has-kv modv="not" k="highway" v="primary"/>
-            <has-kv modv="not" k="highway" v="secondary"/>
-            <has-kv modv="not" k="amenity" v="bus_station"/>
-          </query>
-          <print order="quadtile"/>
-          <query type="node">
-            <around lat="%s" lon="%s" radius="50.0"/>
-            <has-kv k="name"/>
-            <has-kv modv="not" k="highway" v="bus_stop"/>
-          </query>
-          <print order="quadtile"/>
-        </osm-script>
-        """ % (self.lat, self.lon, self.lat, self.lon))
-
-        candidates = []
-
-        # get all node candidates
-        for node in result.get_nodes():
-            if 'name' in node.tags and node.tags["name"] is not None:
-                candidates.append(node)
-
-        # get way node candidates
-        for way in result.get_ways():
-            if 'name' in way.tags and way.tags["name"] is not None:
-                candidates.append(way)
-
-        # leave if no candidates
-        if len(candidates) == 0:
-            # give stop a different name, so we won't search again without refreshing data
-            self.name = "Ponto sem nome"
-            return
-
-        # find closest candidate
-        winner = None
-        winner_distance = sys.maxint
-        for candidate in candidates:
-            if isinstance(candidate, overpy.Way):
-                lat, lon = Stop.get_center_of_nodes(candidate.get_nodes(resolve_missing=True))
-                distance = util.ApproximateDistance(
-                    lat,
-                    lon,
-                    self.lat,
-                    self.lon
-                )
-            else:
-                distance = util.ApproximateDistance(
-                    candidate.lat,
-                    candidate.lon,
-                    self.lat,
-                    self.lon
-                )
-            if distance < winner_distance:
-                winner = candidate
-                winner_distance = distance
-
-        # take name from winner
-        self.name = winner.tags["name"].encode('utf-8')
-
-    @staticmethod
-    def from_relation_member(member):
-        stop_id = member.ref
-
-        if isinstance(member, overpy.RelationNode):
-            stop_type = "node"
-        elif isinstance(member, overpy.RelationWay):
-            stop_type = "way"
-        else:
-            raise RuntimeError("Unknown type: " + str(member))
-
-        return Stop(stop_id, stop_type)
-
+    # TODO: Move over to Fenix implementation
     @staticmethod
     def normalize_name(old_name):
         name = STOP_REGEX.sub(r'\1', old_name)
```

Comment:
Could you please add this here as a temporary fix:
```python
name = name.replace('Terminal Centro', 'TICEN')
```
Somebody has renamed this station and broke the normalization https://www.openstreetmap.org/way/43719878#map=18/-27.59847/-48.55306

This gets us some more trips back.